docs/userguide/networking/get-started-overlay.md

```diff
@@ -314,11 +311,15 @@ to have external connectivity outside of their cluster.
 
 You can try starting a second network on your existing Swarm cluster using Docker Compose.
 
-1. Log into the Swarm master.
+1. Install Docker Compose. If you install on a Mac or Windows with Docker
```

Comment:
Is that true on Windows and Mac? I thought I'd had to explicitly select Compose when installing 1.9 on Windows. Anyway, your suggestion covers all bases so I'll switch to that!